css/interface.css

```diff
@@ -0,0 +1,676 @@
+  display: block;
+  -webkit-user-select: none;
+  padding: 0 0 8px;
+  overflow-y: auto;
+  overflow-x: hidden;
+  margin: 0;
+}
+
+.floor-panels-holder.empty .panel-group,
+.floor-markers-holder.empty .panel-group {
+  display: none;
+}
+
+.panel {
+  opacity: 1;
+  -webkit-transition: opacity 100ms linear;
+}
+
+.panel.focus {
+  opacity: 0.7;
+}
+
+.panel-group .panel {
+  overflow: hidden;
+}
+
+.panel.focus .glyphicon-remove {
+  opacity: 0 !important;
+  -webkit-transition: opacity 50ms linear;
+}
+
+.panel-heading {
+  position: relative;
+  padding: 0;
+}
+
+.panel-heading:hover {
+  cursor: -webkit-grab;
+  cursor: -moz-grab;
+}
+
+.panel-heading h4 {
+  padding: 10px 40px 10px 6px;
+}
+
+.floor-markers-holder .panel-heading:hover {
+  cursor: pointer;
+}
+
+.floor-markers-holder .panel-heading h4 {
+  padding: 10px 40px 10px 10px;
+}
+
+.panel-title {
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  font-size: 14px;
+}
+
+.panel-title .fa {
+  cursor: pointer;
+}
+
+.panel-chevron {
+  position: absolute;
+  top: 0;
+  right: 0;
+  opacity: 0.3;
+  width: 38px;
+  height: 38px;
+  line-height: 38px;
+  text-align: center;
+  -webkit-transition: -webkit-transform 0.25s ease;
+  transition: transform 0.25s ease;
+}
+
+.collapsed .panel-chevron {
+  -webkit-transform: rotate(180deg);
+  transform: rotate(180deg);
+}
+
+.panel-heading .icon-delete {
+  position: absolute;
+  top: 0;
+  right: 38px;
+  opacity: 0;
+  color: #e03629;
+  width: 20px;
+  height: 38px;
+  line-height: 38px;
+  text-align: center;
+}
+
+.panel-heading:hover .icon-delete {
+  opacity: 1;
+}
+
+.panel-heading .screen-reorder-handle {
+  display: block;
+  width: 2.4rem;
+  float: left;
+  margin-right: 6px;
+  text-align: center;
+  opacity: 0.3;
+  line-height: 18px;
+  font-size: 1.2em;
+}
+
+.panel-heading .screen-reorder-handle .fa ~ .fa {
+  margin-left: 2px;
+}
+
+.panel-collapse {
+  opacity: 0;
+}
+
+.panel-collapse.in {
+  opacity: 1;
+  -webkit-transition: opacity 0.1s linear;
+}
+
+.panel-default.sortable-ghost {
+  opacity: 0.5;
+}
+
+.panel-default.sortable-ghost .panel-heading {
+  border: 1px solid rgb(0, 171, 210);
+}
+
+.floor-panel .panel-body label,
+.marker-panel .panel-body label {
+  font-weight: normal;
+}
+
+.selected-folder {
+  margin-top: 5px;
+}
+
+.selected-folder span {
+  font-weight: bold;
+}
+
+.icon-holder {
+  height: 44px;
+  display: flex;
+  align-items: center;
+}
+
+.icon-wrapper {
+  float: left;
+}
+
+.markers-form-group,
+.add-markers-form-group {
+  margin-top: 40px;
+}
+
+.add-markers-form-group .text-danger {
+  margin-top: 5px;
+}
+
+/* COLORPICKER */
+[colorpicker-component] {
+  width: 100%;
+}
+
+[colorpicker-component] .input-group-addon {
+  border-top-left-radius: 6px;
+  border-bottom-left-radius: 6px;
+  width: 46px;
+  position: relative;
+  overflow: hidden;
+}
+
+[colorpicker-component] .input-group-addon + .form-control:not(:first-child):not(:last-child) {
```

Comment:
Why would something be `:first-child` if the selector segment follows a `+` (meaning it comes after something else)?